trace/tests/unit/ext/django/app/views.py

```diff
@@ -20,12 +20,17 @@
 from opencensus.trace import config_integration
 
 import mysql.connector
+import psycopg2
+
 import time
 import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+"/../../../.."))
```

Comment:
We really, really, want to avoid modifying `sys.path` at runtime if at all possible.  What is the underlying issue?

Reply:
Leave it for now as discussed.